### PR TITLE
Improve error around the repo_name argument in support_panel()

### PR DESCRIPTION
**Pull request by cjrace**
Making it more explicit in the documentation and error message why the repo_name field is erroring.

### CODE REVIEW

File: R/support_panel.R [hunk 1]
```diff
@@ -71,8 +72,10 @@ support_panel <- function(
 
   if (is_valid_repo_name(repo_name) == FALSE) {
     stop(
-      "Please ensure the repo_name argument points to a repository on the
-      dfe-analytical-services GitHub area."
+      "Please ensure the repo_name argument is a valid URl to a repository on
```
Comment (jen-machin) [inline R/support_panel.R:75]: think this is a lower case l here and not upper case?

Comment (cjrace) [reply R/support_panel.R]: Excellent spot! Have fixed that and pushed a commit up for it now

**Comment by rmbielby** [comment]
> One thought I had is whether we should rename the argument from repo_name to repo_url to help make it clearer. Though wary of how much this might be a breaking change anyone who has adopted it - @rmbielby, what do you think?

It's not used too widely so far, so we could probably fairly easily go round and do the updates. And it'll give a pretty clear error when it does break.

**Comment by rmbielby** [comment]
Just added some fixes to the links as they were all broken from us trying to meet line-length constraints!

**Review by rmbielby — APPROVED** [review]

**Pull request merged**